diy/cases.md

```diff
@@ -128,7 +128,7 @@ The SlimeVR community has built a huge number of cases for style, internals and
 ## Arcturus by Lixulia
 Design by Lixulia
 
-![Case](https://raw.githubusercontent.com/Lixulia/Arcturus/main/renders/renderforarcturus.PNG) {: .caseImage}
+![Case](../assets/img/renderforarcturusnew.png)
```

Comment:
```suggestion
![Case](../assets/img/renderforarcturusnew.png) {: .caseImage}
```

Reply:
that doesn't seem to work on my side! it just adds the text of .caseImage below

Reply:
Because vscode doesn’t use it, but it works with Jekyll